### DISCUSSION TITLE
Remove playlist status icon

### DISCUSSION THREAD

**Discussion by SunsetTechuila** [nimsandu/spicetify-bloom#216] [Ideas]
It has so many problems: it's out of style, it prevents you from opening a playlist, it disappears on pause affecting internal icon padding, on liked tracks and podcasts it is located on the edge, but on playlists it is centered, it's too small on the big screens and too big on the small. At the same time it's almost useless! So maybe better to remove it completely? What do you think @nimsandu?

**Comment by SunsetTechuila** [comment]
![image](https://user-images.githubusercontent.com/115353812/226138813-20a37425-1839-46a5-8c83-86fe825fd234.png)
I'm speaking about this thing btw

**Comment by nimsandu** [comment]
@SunsetTechuila I thought about it. 
It's less useful and results user confusions even though it's supposed to be useful. 
So I would say, it's time to remove it @SunsetTechuila
